ui/src/app/pages/admin/admin-institution-edit.tsx

```diff
@@ -201,22 +240,24 @@ export const AdminInstitutionEdit = withUrlParams()(class extends React.Componen
   // b) email address/Domain are not valid
   // c) Required fields are not empty
   disableSave(errors) {
-    return this.validateRequiredFields() || errors || this.fieldsNotEdited()
+    return this.hasInvalidFields() || errors || this.fieldsEdited()
       || this.state.invalidEmailAddress || this.state.invalidEmailDomain;
   }
 
   async saveInstitution() {
     const {institution, institutionMode} = this.state;
-    if (institution) {
-      if (institution.duaTypeEnum === DuaType.MASTER) {
-        institution.emailAddresses = [];
-      } else {
-        institution.emailDomains = [];
-      }
-      if (institution.organizationTypeEnum !== OrganizationType.OTHER) {
-        institution.organizationTypeOtherText = null;
+    const rtRequirement = getRegisteredTierRequirement(institution);
+    if (institution && rtRequirement) {
```

Comment:
Seems like the existence of `institution` should be a precondition for this running this method - are we allowing you to create or update a null institution?

Would check this and probably return above, or just ignore it and let the errors propagate. The check here is pointless right now since you'll just NPE at L257 if this is null/undefined.